design.config.js

```diff
@@ -0,0 +1,37 @@
+const path = require('path')
+
+const isDev = process.env.NODE_ENV !== 'production'
+
+const icons = ['ab-testing']
+
+if (isDev) icons.push('check')
+
+const iconsRoot = path.join(
+  path.dirname(require.resolve('@mdi/svg/package.json')),
+  './svg'
+)
+
+const config = {
+  icons: {
+    path: iconsRoot,
+    files: icons,
+  },
+
+  output: {
+    path: path.join(__dirname, 'public'),
+    publicPath: '/',
+    icons: {
+      files: 'icons',
+      sprite: 'icons.svg',
+    },
+  },
+
+  svgo: {
+    plugins: {
+      removeViewBox: false,
+      removeDimensions: true,
+    },
+  },
```

Comment:
When I said, your `design.config.js` can be simpler, I meant this part is implicit. I doubt it will ever change so let's keep it out of the config and even out of the documentation (if we would have any 😁). It an internal API.